src/components/pages/LandingPage/LandingPage.js

```diff
@@ -72,23 +78,27 @@ function LandingPage() {
             recidivism and reentry on prisonpolicy.org/recidivism the ultimate
             guide on prisoninsight.com
           </p>
-          <button className="learn">Learn more</button>
         </div>
         <div className="right2">
           <h3 className="subHead2">Unemployment and Poverty</h3>
           <p className="text1">
             Employment has long been recognized as having a negative
             correclation with crime (Uggen, 1999; Uggen et al., 2005)
             <br />
+            <br />
```

Comment:
I love your implementation of each section. Although looking at the line above I am a bit confused as to why you added two line breaks here. Can you explain the purpose to me?

Reply:
These are necessary visual breaks to match Figma's design :)